indico/web/templates/forms/_form.html

```diff
@@ -178,4 +183,18 @@
             {% endif %}
         {% endif %}
     {% endcall %}
+    {% if form_header_kwargs.get('id') == 'badge-settings-form' %}
+        <script>
+            document.getElementById('form-submit-button').addEventListener('click', event => {
+                const button = event.target;
+                const formElement = document.getElementById('{{ form_header_kwargs['id'] }}');
+                const request = new XMLHttpRequest();
```

Comment:
XHR is pretty obsolete, usually we use axios (`indicoAxios` is available for import in JS files), so if you really need to send any requests from code in HTML, I'd use `fetch()` for it. But I'd prefer if we did not need this code here at all.